librz/core/agraph.c

```diff
@@ -2406,6 +2429,28 @@ static int get_bbnodes(RzAGraph *g, RzCore *core, RzAnalysisFunction *fcn) {
 		char *title = get_title(bb->addr);
 		RzANode *u = rz_agraph_get_node(g, title);
 		RzANode *v;
+
+		RzListIter *it;
+		RzBinTrycatch *tc;
+		rz_list_foreach (exception_scopes, it, tc) {
+			if (bb->addr == tc->handler) {
+				continue;
+			}
+			if (!rz_itv_overlap2((RzInterval){ bb->addr, bb->size }, tc->from, tc->to - 1)) {
+				continue;
+			}
+			RzAnalysisBlock *catch_bb = rz_analysis_get_block_at(bb->analysis, tc->handler);
+			if (!catch_bb || catch_bb->addr == bb->jump || catch_bb->addr == bb->fail) {
+				continue;
+			}
+			title = get_title(catch_bb->addr);
+			v = rz_agraph_get_node(g, title);
+			free(title);
```

Comment:
```suggestion
			RZ_FREE(title);
```
Use RZ_FREE when a pointer is reused.
RZ_FREE will invoke `free` and also set the freed pointer to `NULL`.